varya/assets/js/main-navigation.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * File main-navigation.js.
+ *
+ * Required to open and close the mobile navigation.
+ */
+
+( function() { 
+	document.getElementById( "toggle-menu" ).onclick = function() {
```

Comment:
This might be overkill, but I think this will cause a TypeError if for some reason the page loads without an element of id `toggle-menu`. 

We could do something like: 
```js
document.getElementById( "toggle-menu" ) ? 
document.getElementById( "toggle-menu" ).onclick = function() { ... } : null; 
```

Reply:
Alternatively or maybe in addition to this, we should add a conditional [here](https://github.com/Automattic/themes-workspace/pull/56/files#diff-0024df2bc4f430cba2d7ab5c0240ecabR293) that only enqueues this script when the `menu-1` location is actually active. That way, we only enqueue this script when its needed which will ensure that the `#toggle-menu` ID will be available in the dom.